pkg/kubectl/cmd/get.go

```diff
@@ -127,7 +128,8 @@ func NewCmdGet(f cmdutil.Factory, out io.Writer, errOut io.Writer) *cobra.Comman
 	}
 	cmdutil.AddPrinterFlags(cmd)
 	cmd.Flags().StringP("selector", "l", "", "Selector (label query) to filter on, supports '=', '==', and '!='.(e.g. -l key1=value1,key2=value2)")
-	cmd.Flags().BoolP("watch", "w", false, "After listing/getting the requested object, watch for changes.")
+	cmdutil.AddIncludeUninitializedFlag(cmd)
+	cmd.Flags().BoolP("watch", "w", false, "After listing/getting the requested object, including uninitialized single one, watch for changes.")
```

Comment:
How about "After listing/getting the requested object, watch for changes. Uninitialized objects are excluded if no object name is provided."